src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/ContributorInviteRequest.cs

```diff
@@ -0,0 +1,13 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
+{
+    public class ContributorInviteRequest
+    {
+        public string Email { get; set; }
```

Comment:
Added `[Required]` to both and tested sending requests without the fields, and got appropriate errors:

![image](https://user-images.githubusercontent.com/44812484/146540606-d845dc52-6127-4054-a432-46cf461ab913.png)

Reply:
I think that's right and what I was expecting but the annoying thing is it leaves a warning that those fields are potentially uninitialised .

![image](https://user-images.githubusercontent.com/4147126/146917504-a68f78b9-ab0e-415f-a3e4-4478a39254b8.png)

If we fix those warning by initialising them (i.e. with string.empty for the email and a new list for the Release id's) then the `[Required]` validation will never fire.

Alternatively if we fix those warning by making them nullable then every place using them has to handle them potentially being null, when in reality we know that because of the `[Required]` validation that null should never be a possibility.

I think I'll raise this with the other developers to try and reach a consensus on what we should do here.

Reply:
We just discussed this and decided that we will do a test omitting the fields in the API request after initialising them to default values i.e. with string.empty for the email and a new list for the Release id's, and double check whether the `[Required]` validation fires or not.

If it doesn't then we can initialise them with `null!` so the warnings disappear but the validation should still fire.

Reply:
Made changes to `ContributorInviteViewModel` and `UserInviteViewModel` as discussed in Slack